src/pages/TradeV3/TradeConfirmation.tsx

```diff
@@ -98,60 +102,121 @@ export const TradeConfirmation: FC<{ setVisibility: (bool: boolean) => any; take
+  //       <ROW>
+  //         <span>Fee (0.1%)</span>
+  //         <span className="value">{fee} USDC</span>
+  //       </ROW>
+  //       <ROW>
+  //         <span>Total Cost</span>
+  //         <span tw="mb-7" className="value">
+  //           {total} USDC
+  //         </span>
+  //       </ROW>
+  //       {/* <ROW>
+  //         <span>Est. Liquidation Price</span>
+  //         <span className="value">$14.9628</span>
+  //       </ROW> */}
+  //     </div>
+  //     <Button
+  //       onClick={() => handleClick()}
+  //       width="100%"
+  //       loading={isLoading}
+  //       disabled={isLoading}
+  //       height={checkMobile() ? '45px' : '50px'}
+  //       cssStyle={cssStyle}
+  //     >
+  //       {order.side === 'buy' ? 'Long' : 'Short'} {Number(order.size).toFixed(3)} {symbol}
+  //     </Button>
+  //   </WRAPPER>
+  // )
   return (
-    <WRAPPER>
-      <div tw="mt-[30px] mb-7 sm:my-0">
-        <ROW>
-          <span>Order Type</span>
-          <span className="value">{order.display === 'limit' ? 'Limit' : 'Market'}</span>
-        </ROW>
-        <ROW>
-          <span>Trade Size</span>
-          <span className="value spacing">
-            {Number(order.size).toFixed(5)} {symbol}
-          </span>
-        </ROW>
-        <ROW>
-          <span>Est. Entry Price</span>
-          <span className="value">${order.price}</span>
-        </ROW>
-        <ROW>
-          <span>Est. Price Impact</span>
-          <span className="value">{totalPriceImpact.toFixed(4)}%</span>
-        </ROW>
-        <ROW>
-          <span>Slippage Tolerance</span>
-          <span className="value spacing">{'0.1%'}</span>
-        </ROW>
-        <ROW>
-          <span>Trade Notional Size</span>
-          <span className="value">{notionalValue} USDC</span>
-        </ROW>
-        <ROW>
-          <span>Fee (0.1%)</span>
-          <span className="value">{fee} USDC</span>
-        </ROW>
-        <ROW>
-          <span>Total Cost</span>
-          <span tw="mb-7" className="value">
-            {total} USDC
-          </span>
-        </ROW>
-        {/* <ROW>
-          <span>Est. Liquidation Price</span>
-          <span className="value">$14.9628</span>
-        </ROW> */}
-      </div>
-      <Button
-        onClick={() => handleClick()}
-        width="100%"
-        loading={isLoading}
-        disabled={isLoading}
-        height={checkMobile() ? '45px' : '50px'}
-        cssStyle={cssStyle}
+    <Dialog open={open} onOpenChange={setVisibility}>
+      <DialogOverlay className={'z-[961]'} />
+      <DialogContent
+        size="md"
+        placement={checkMobile() ? 'bottom' : 'default'}
+        className={cn('z-[962] sm:w-[100vw] min-md:w-[500px] sm:bottom-0 h-[346px] pt-3 flex flex-col gap-0')}
       >
-        {order.side === 'buy' ? 'Long' : 'Short'} {Number(order.size).toFixed(3)} {symbol}
-      </Button>
-    </WRAPPER>
+        <DialogHeader className={`space-y-0 w-full h-[58px] px-2.5 text-left `}>
```

Comment:
would specififying some prebuilt styles for header/footer/body help prevent the manual setting we do?